src/Dommel/Get.cs

```diff
@@ -33,13 +34,14 @@ public static partial class DommelMapper
         /// <param name="connection">The connection to the database. This can either be open or closed.</param>
         /// <param name="id">The id of the entity in the database.</param>
         /// <param name="transaction">Optional transaction for the command.</param>
+        ///  <param name="cancellationToken">Optional cancellationToken for the command.</param>
         /// <returns>The entity with the corresponding id.</returns>
-        public static async Task<TEntity?> GetAsync<TEntity>(this IDbConnection connection, object id, IDbTransaction? transaction = null)
+        public static async Task<TEntity?> GetAsync<TEntity>(this IDbConnection connection, object id, IDbTransaction? transaction = null, CancellationToken? cancellationToken = null)
```

Comment:
I think it's a best practice to use this pattern, rather than a nullable:

```suggestion
        public static async Task<TEntity?> GetAsync<TEntity>(this IDbConnection connection, object id, IDbTransaction? transaction = null, CancellationToken cancellationToken = default)
```

Can you apply this everywhere? You can also remove the `?? default` later then.